config/application.rb

```diff
@@ -18,6 +18,7 @@ class Application < Rails::Application
 
     # Google Discovery Engine configuration
     config.discovery_engine_serving_config = ENV.fetch("DISCOVERY_ENGINE_SERVING_CONFIG")
+    config.discovery_engine_datastore = ENV.fetch("DISCOVERY_ENGINE_DATASTORE")
```

Comment:
Do you need to add this to the README?

Wider question on past choices, did you consider having these as optional for at least the test environment? feels like it could be a bit painful to need this set to run the tests (also painful for dev but I guess the app is pretty useless without them)

Reply:
(oh and for deployment, this variable is actually already present but unused in the app's GCP environment configuration)

Reply:
Yeah, can understand the can of worms.

Only bit about this comment still outstanding is the README where it lists the env vars: https://github.com/alphagov/search-api-v2/blob/dcce01b12fb3cb300e83a797bb423c9de099bd1a/README.md#L16

Having now paid closer attention to the README, and seen that all it says is that GOV.UK Docker sets them up, perhaps it's better to delete the listing of env vars from the README and/or to say GOV.UK Docker has the appropriate env vars configured for working with the search-api-v2 GCP integration project.

Reply:
Good shout – definitely better to have that in fewer places!